File: 卡码/KamaCoder/Python/test09.py
#9. 奇怪的信
import sys

def main():
	'''
	while True:
		try:
			n=input()
			print(sum(list(map(int,filter(lambda x:int(x)%2==0,n)))))
			print()
		except:
			break
	'''
	for line in sys.stdin:
		print(sum(list(map(int,(filter(lambda x:int(x)%2==0,line[:-1] if line[-1]=='\n' else line))))))
		print()

if __name__ == '__main__':
	main()

File: 卡码/KamaCoder/Js/test09.js
//9. 奇怪的信
const rl=require("readline").createInterface({input:process.stdin});
const iter=rl[Symbol.asyncIterator]();
const readline=async ()=>(await iter.next()).value;

async function main(){
  while(line=await readline()){
    const nums=line.split("").map(Number);
    let res=0;
    nums.forEach(num=>{
      if(num%2==0) res+=num;
    });
    console.log(res);
    console.log();
  }
}

main();